typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/endItemDecoratorStrategy.ts

```diff
@@ -35,6 +36,8 @@ import { GraphNode } from "./helpers/graph";
  * ```
  */
 export default class EndItemDecoratorStrategy implements CanvasItemDecoratorStrategy {
+	/** Adjustment to canvas [x,y] coordinates to compensate for item icon anchor position */
+	public static get END_ITEM_OFFSET(): [number, number] { return [40, -10]; }
```

Comment:
public static END_ITEM_OFFSEt = [40,-10]

Reply:
Didn't want the array to be altered accidentally